02-wrangling.Rmd

```diff
@@ -543,8 +543,6 @@ The `map_*` functions are generally quite useful for solving problems involving
 of a data frame; `map_*` functions can be used to apply functions to elements of a vector or list, and even to lists of data frames, or nested data frames.
 
 ## Additional resources
-Grolemund & Wickham's [R for Data Science](https://r4ds.had.co.nz/) has a number of useful sections that provide additional information:
-
-- [Data transformation](https://r4ds.had.co.nz/transform.html)
-- [Tidy data](https://r4ds.had.co.nz/tidy-data.html)
-- [The `map_*` functions](https://r4ds.had.co.nz/iteration.html#the-map-functions)
+- The [`dplyr` page on the tidyverse website](https://dplyr.tidyverse.org/) is where you should look if you want to learn more about the functions in this chapter, the full set of arguments you can use, and other related functions. The site also provides a very nice cheat sheet that summarizes many of the data wrangling functions from this chapter.
+- [R for Data Science](https://r4ds.had.co.nz/) has a few chapters related to data wrangling that go into more depth than this book. For example, the [tidy data](https://r4ds.had.co.nz/tidy-data.html) chapter covers the concept of tidy data, missing values, `pivot_longer`/`pivot_wider`, and additional wrangling functions (like `unite` and `separate`); the [data transformation](https://r4ds.had.co.nz/transform.html) chapter covers `select`, `filter`, `arrange`, `mutate`, and `summarize`; and the [`map_*` functions](https://r4ds.had.co.nz/iteration.html#the-map-functions) chapter provides more about the `map_*` functions.
```

Comment:
@leem44 per our slack convo, look here for possible mistakes I made in discussing extra funcs!